doc/content/documentation/array-table-and-struct.md

```diff
@@ -107,6 +123,48 @@ An array can also be used as a stack. Therefore, the `push`, `peek` and `pop` fu
   (pop arr)) # Evaluates to 1, arr is empty @[]
 ```
 
+## Set operations
+
+A set provide very specific functions described in set theory. Such as `union`, `intersection`, `difference` and `symmetric-difference`.
```

Comment:
A set provide**s** very specific functions **that are** described in the set theory. **These are** `union`,...